doc/classes/Array.xml

```diff
@@ -40,6 +40,7 @@
 		[/codeblocks]
 		[b]Note:[/b] Arrays are always passed by reference. To get a copy of an array that can be modified independently of the original array, use [method duplicate].
 		[b]Note:[/b] Erasing elements while iterating over arrays is [b]not[/b] supported and will result in unpredictable behavior.
+		[b]Packed arrays versus typed arrays:[/b] Packed arrays are generally faster to iterate on and modify compared to a typed array of the same type (e.g. [code]Array[int][/code]). Typed arrays are in turn faster to iterate on and modify than untyped arrays. As a downside, packed arrays are less flexible as they don't offer as many convenience methods such as [method Array.map].
```

Comment:
I have seen the claim that typed arrays are slightly **slower** than untyped ones. At least when adding elements, since the value needs to be validated. At the same time, element operations can indeed be faster if GDScript uses typed instructions. Are there any benchmarks?

Reply:
With https://github.com/godotengine/godot-benchmarks, typed arrays are consistently faster than untyped arrays on my end.